src/main/java/hec/stats/MinComputable.java

```diff
@@ -0,0 +1,17 @@
+package hec.stats;
+
+public class MinComputable implements Computable{
+    @Override
+    public float compute(float[] values){
+        //calculate the min of values
+        float returnvalue = 99999999999.00f;
```

Comment:
Suggest using the Arrays.sort to cut down on the number of lines. Then select the [0] to get the first value.

Also add test to show your code is doing what it says

Reply:
Do you think sorting will be faster than iterating through the array one time?

Good idea on adding a test! we should definitely do that

Reply:
Yes, appears sorting is less efficient than iterating.  Cleaner code?

Reply:
more understandable code is always a good thing. We need to be concerned with this processing for efficiency though. The amount of data we are dealing with is huge! millions of milliseconds add up.

We can change the code to sort to make it more readable and start testing, but we will want to optimize for speed as we move forward with the project. Thoughts?

Reply:
Agreed, especially with the number of values that need to be dealt with.